src/Main.elm

```diff
@@ -1,32 +1,41 @@
 module Main exposing (..)
 
-import Html exposing (Html, text, div, h1, img)
-import Html.Attributes exposing (src)
-
+import Html exposing (Html, text, div, h1, img, button)
+import Html.Events exposing (onMouseDown, onMouseUp)
+import Ports exposing (..)
+import Types exposing (..)
 
 ---- MODEL ----
 
+loadNoteSample : Note -> Cmd msg
+loadNoteSample note =
+    loadSample (toString note, "samples/" ++ (toString note) ++ ".mp3")
```

Comment:
I noticed 'toString' in a few places threw the code. What could we do to minimize its use.

Reply:
There is no alternatives. In this case we build a URL and send it to the JS port.